tutorials/geckoview-quick-start.md

```diff
@@ -82,13 +82,42 @@ Now you're set up and ready to go.
 
 One you have got GeckoView building and running, you will want to start contributing. There is a general guide to [Performing a Bug Fix for Git Developers](contributing-to-mc.md) for you to follow. To contribute to GeckoView specifically, you will need the following additional information.
 
-It is advisable to run your tests before submitting your patch. You can do this using Mozilla's `try` server. To submit a GeckoView patch to `try` before submitting it for review, type:
+### Updating the changelog and API documentation
+
+If the patch that you want to submit changes the public API for GeckoView, you must ensure that the API documentation is kept up to date. To check whether your patch has altered the API, run the following command.
+
+```bash
+./mach android api-lint
 ```
+
+The output of this command will inform you if any changes you have made break the existing API. Review the changes and, if they are correct, update your api.txt to conform to your changed by running:
```

Comment:
First of all: thanks for writing this up!

I have a question: the `./mach android api-lint` already gives you some information similar to this, and it's context aware so that e.g. `apiUpdateFileWithGeckoBinariesDebug` command will always be the correct one (it has changed already once) and similarly the `CHANGELOG.md` location will always be correct.

Do you think it would be worth it to make the command error messages better and just say here "run `./mach android api-lint` and follow instructions"?

Reply:
You are probably right. The thing I wanted to highlight mostly was the format to use for the changelog entries, so we can drop everything but that.